modules/gapi/samples/privacy_masking_camera.cpp

```diff
@@ -0,0 +1,215 @@
+#include <chrono>
+#include <iostream>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/imgproc.hpp>
+#include <opencv2/gapi/infer.hpp>
+#include <opencv2/gapi/render.hpp>
+#include <opencv2/gapi/infer/ie.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+#include <opencv2/gapi/streaming/cap.hpp>
+#include <opencv2/highgui.hpp>
+
+const std::string about =
+    "This is an OpenCV-based version of Privacy Masking Camera example";
+const std::string keys =
+    "{ h help |   | print this help message }"
+    "{ input  |   | Path to an input video file }"
+    "{ platm  |   | IE vehicle/license plate detection model IR }"
+    "{ platw  |   | IE vehicle/license plate detection model weights }"
+    "{ platd  |   | IE vehicle/license plate detection model device }"
+    "{ facem  |   | IE face detection model IR }"
+    "{ facew  |   | IE face detection model weights }"
+    "{ faced  |   | IE face detection model device }"
```

Comment:
It would be nice to have default values for model-related parameters and/or some instructions how to get them

Reply:
Not sure what may be defaults for model paths, here you mean defaults for devices?

Reply:
model ~~paths~~ names at first
devices - `CPU` by default can be used too (with example of possible values in parameter's description: CPU/GPU/VPU/etc)

Goal: get working sample with minimal command line (perhaps with copying of model files...).

Also you may add some examples into `about` string.

Reply:
Kept path so far

Reply:
Use `name` of the model by default (which just works if models are already in current working directory).
User may override them with own path.

Main idea of that is showing what User should specify here.

Reply:
What is the name of the model?

Reply:
Some name of model which can be used with this sample (which knows label=2), like `vehicle-license-plate-detection-barrier-0106` from OMZ.

Reply:
So then you want this sample to append .xml automatically?
I found this counter-intuitive if the model is placed in some other directory. Then the "model name" effectively becomes a model path

Reply:
Renamed its description to "Path to <..>"

Reply:
> append .xml automatically

No.

---

Path of which model user should write here?
How many models users should try before they run demo successfully?
Select one that works successfully with this sample and write here as "default" (filename only).

> or some instructions how to get them

This is that I want to see here. "Model name" (from OMZ) is an instruction.

Reply:
Ok so now I've finally got it :)

Reply:
@alalek done

Reply:
What is a purpose of strings below?

```
G_API_NET(VehLicDetector, <cv::GMat(cv::GMat)>, "vehicle-license-plate-detector");
G_API_NET(FaceDetector,   <cv::GMat(cv::GMat)>,                  "face-detector");
```

Model names?
If they are just internal names of operations, then my suggestion is:

```
    "{ platm  | vehicle-license-plate-detection-barrier-0106.xml | Path to OpenVINO IE model .xml file (plate detection) }"
    "{ platd  | CPU   | Target device for plate detection model (e.g. CPU, GPU, VPU, ...) }"
    "{ facem  | face-detection-adas-0001.xml | Path to OpenVINO IE model .xml file (face detection) }"
    "{ faced  | CPU   | Target device for face detection model (e.g. CPU, GPU, VPU, ...) }"
```

Reply:
> If they are just internal names of operations

This

> then my suggestion is

OK

Reply:
@alalek done